src/types/redis_stream.cc

```diff
@@ -1698,4 +1698,91 @@ rocksdb::Status Stream::SetId(const Slice &stream_name, const StreamEntryID &las
   return storage_->Write(storage_->DefaultWriteOptions(), batch->GetWriteBatch());
 }
 
+rocksdb::Status Stream::GetPendingEntries(StreamPendingOptions &options, StreamGetPendingEntryResult &pending_infos,
```

Comment:
Can we pass output parameters by pointers?

Reply:
We can, but why? I'm not fully understand the necessity of using pointer.

Reply:
@Yangsx-1 It's not a requirement from my side :) I just saw some conventions that output parameters go at the end of the list and are passed by a pointer. Maybe it has its roots in C. However, references are safer since you can't dereference a null pointer.